pkg/runtime/rules.go

```diff
@@ -26,5 +26,10 @@ func (e *Executor) initRules() error {
 	if len(configReader.GetRules().SkipRules) > 0 {
 		e.skipRules = append(e.skipRules, configReader.GetRules().SkipRules...)
 	}
+
+	// specify severity of violations to be reported
+	if len(configReader.GetSeverity().Level) > 0 {
```

Comment:
Does this introduces a scope for a bug related to validation of severity level? Since the validation happens before init is called, e.severity will be assigned with a value in the config file and would not be validated.

Reply:
I think, there is a valid point. Here we are assigning the value read from the config file, which will override the value set on the CLI.

Reply:
@patilpankaj212 @kanchwala-yusuf  PTAL at the executor.go function call flow. This has been resolved there. 

https://github.com/accurics/terrascan/blob/8373b84d1df6dfb84fe7c8c756e1d1dbdf983937/pkg/runtime/executor.go#L47

Reply:
got it. skip rules, scan rules and severity is update with config file values before the executor init is called.